includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -244,7 +244,7 @@ public function register_routes() {
 	 */
 	public function get_settings() {
 		$is_upe_enabled               = WC_Stripe_Feature_Flags::is_upe_checkout_enabled();
-		$available_payment_method_ids = $is_upe_enabled ? $this->gateway->get_upe_available_payment_methods() : WC_Stripe_Helper::get_legacy_available_payment_method_ids();
+		$available_payment_method_ids = $is_upe_enabled ? $this->gateway->get_upe_available_payment_methods( true ) : WC_Stripe_Helper::get_legacy_available_payment_method_ids();
 		$ordered_payment_method_ids   = $is_upe_enabled ? WC_Stripe_Helper::get_upe_ordered_payment_method_ids( $this->gateway ) : $available_payment_method_ids;
 		$enabled_payment_method_ids   = $is_upe_enabled ? $this->gateway->get_upe_enabled_payment_method_ids( true ) : WC_Stripe_Helper::get_legacy_enabled_payment_method_ids();
 
```

Comment:
I have one concern here: we're forcing two refreshes of the PMC from Stripe when `$is_upe_enabled`, as we're supplying `$force_refresh = true` to both `$this->gateway->get_upe_available_payment_methods()` and `$this->gateway->get_upe_enabled_payment_method_ids( true )`. We should only trigger one re-fetch.
```suggestion
		// Note that we specify the $force_refresh parameter for this fetch only, and not when fetching enabled payment methods below.
		$available_payment_method_ids = $is_upe_enabled ? $this->gateway->get_upe_available_payment_methods( true ) : WC_Stripe_Helper::get_legacy_available_payment_method_ids();
		$ordered_payment_method_ids   = $is_upe_enabled ? WC_Stripe_Helper::get_upe_ordered_payment_method_ids( $this->gateway ) : $available_payment_method_ids;
		$enabled_payment_method_ids   = $is_upe_enabled ? $this->gateway->get_upe_enabled_payment_method_ids( false ) : WC_Stripe_Helper::get_legacy_enabled_payment_method_ids();

```

Reply:
Refetching only once would be better. 👍 

Fixed in [d1453bf](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4252/commits/d1453bfa484eab5f6a0f6d232c975e37b688a621) and [f632a30](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4252/commits/f632a30e095d580eb22809eba2e51082e7837637)